front/src/Components/ConsoleGlobalMessageManager/InputTextGlobalMessage.svelte

```diff
@@ -35,19 +34,39 @@
     export let game: Game;
     export let gameManager: GameManager;
 
-    let gameScene = gameManager.getCurrentGameScene(game.scene.getScene(LoginSceneName));
+    let gameScene = gameManager.getCurrentGameScene(game.findAnyScene());
```

Comment:
So we change of idea? It could be better to get LoginSceneName or other basic scene that the first in scene list?

Reply:
I quite like this idea, but....

We could make this even simpler!

I mean, `game.findAnyScene()` could be private, and we could call it directly from the gameManager.

This way, we could simply write: `gameManager.getCurrentGameScene()` which makes a lot more sense

Reply:
`game.findAnyScene()` is a function of `game` not `gameManager`, so it couldn't be private even if we call it from `gameManager`.

Plus, `gameManager` doesn't have an object `game` in himself, so the best that we could do is `gameManger.getCurrentGameScene(game)` or we could pass the `game` to the `gameManager` when we create it.